package.json

```diff
@@ -92,12 +90,16 @@
     "mocha": "^7.0.0",
     "mv": "^2.1.1",
     "ncp": "^2.0.0",
+    "null-loader": "^3.0.0",
     "prettier": "^1.18.2",
     "proxyquire": "^2.0.0",
     "sinon": "^9.0.0",
     "source-map-support": "^0.5.9",
+    "ts-loader": "^6.2.1",
     "typescript": "3.6.4",
```

Comment:
Can we use `"typescript": "^3.8.3"` as we do for other libraries?

Reply:
As discussion with @feywind, upgrade typescript `^3.8.3` will happen when pubsub is ready for deprecate Node 8 and upgrade gts@2.0.0.